packages/doenetml-worker/lib-doenetml-core/src/dast/ref_expand.rs

```diff
@@ -146,7 +146,8 @@ impl Expander {
         }
     }
 
-    /// Remove all `extend` attributes from the `FlatRoot` and replace them with the index of their referent.
+    /// Remove all `extend` attributes from the `FlatRoot` and replace them
+    /// an extending field that is a ref_resolution with the index of their referent.
```

Comment:
This comment doesn't make sense to me

Reply:
Hah, that's progress, as I didn't understand the original comment! I missed the preposition "with" in the comment; not sure if that caused confusion.

Reply:
With the word `with` it makes more sense :-). I think it would be clearer to say
``replace them with `extending: RefResolution` where the `RefResolution` contains the index of the referent`` (or something like that with the correct variable names...)